playbooks/roles/streisand-mirror/vars/openconnect.yml

```diff
@@ -15,12 +15,20 @@ openconnect_source_url: "https://d1shxn4vnb0yj3.cloudfront.net/{{ openconnect_so
 openconnect_source_checksum: "sha256:facf695368dc4537a6a30e2147be90b1d77ee3cb2d269eaef070b6d9ddab70f2"
 
 # Windows
-openconnect_windows_version: "v1.3"
-openconnect_windows_filename: "openconnect-installer.exe"
+openconnect_windows_version: "v1.5.1"
+openconnect_windows_filename: "openconnect-gui-1.5.1-win32.exe"
 openconnect_windows_href: "{{ openconnect_mirror_href_base }}/{{ openconnect_windows_filename }}"
 openconnect_windows_url: "https://github.com/openconnect/openconnect-gui/releases/download/{{ openconnect_windows_version }}/{{ openconnect_windows_filename }}"
-openconnect_windows_checksum: "sha256:894ce16fbfbb4be0f205dc5bc55323b5d99306e81672ad5d631bd9bab0d586b1"
+openconnect_windows_checksum: "sha256:65ea336108640b44eb16192c7d357953895ce9f61f5ee77d124e56e98714cc7e"
+
+# macOS
```

Comment:
Can you add a brief comment here about why this section is commented out? I would even throw in a link to https://github.com/openconnect/openconnect-gui/issues/160 so that it's easier to spot when this can be uncommented.